docs/content/mobile/ios/setup.md

```diff
@@ -4,10 +4,9 @@
 
 > **Note:** iOS development is only supported on macOS.
 
-  1. Follow the steps in [Get the dependencies]({{urlRoot}}/setup-and-installing) and additionally install **iOS build support** for Unity.
-  1. Install iOS prerequisites
-    * [Xcode](https://developer.apple.com/xcode/)
-      * Once installed, open Xcode. It might prompt you to accept user agreements.
```

Comment:
does this not happen?

Reply:
Sometimes. I don't think we need to tell people that they need to accept Apple agreements to use Apple products though.